PowerPointLabs/PowerPointLabs/ActionFramework/NarrationsLab/RecordNarrations/RecordNarrationsActionHandler.cs

```diff
@@ -14,19 +15,14 @@ class RecordNarrationsActionHandler : ActionHandler
     {
         protected override void ExecuteAction(string ribbonId)
         {
-            //TODO: This needs to improved to stop using global variables
```

Comment:
Any reason for removing this? The `this.GetCurrentPresentation().Presentation` below looks like accessing a global variable, and its still required.

Reply:
It is an extension wrapper that hides the global implementation on another level. I can add it back.